examples/vercel-ai-chatbot/lib/hooks/use-vellum-chat.tsx

```diff
@@ -0,0 +1,205 @@
+import { nanoid } from 'nanoid'
```

Comment:
This file is my pitch for why we should start publishing a `@vellum/react` package